build/build.py

```diff
@@ -98,6 +98,10 @@ def __init__(self):
             '--build-for-snap', dest='build_for_snap', action='store_true',
             help='Only build components and features those required for the Ubuntu Core snap'
         )
+        self.parser.add_argument(
+            '--install-prefix', dest='install_prefix', type=str,
```

Comment:
Rename the option to --search-prefix. Install prefix is the next change to come.